.github/workflows/ebmc-packages.yaml

```diff
@@ -1,10 +1,11 @@
 on:
   release:
-    types: [created]
+    types: [published]
```

Comment:
I think it would be better to use tags as the trigger, else you end up publishing a release but don't yet have artifacts in place at the time of publication (and instead hope for them to be created soonish). See https://github.com/model-checking/kani/blob/main/.github/workflows/release.yml#L8-L15 for triggers that appear to work well for us.

Reply:
Yes, will change once this flow works.